nilearn/region.py

```diff
@@ -9,6 +9,7 @@
 
 import numpy as np
 from scipy import linalg, ndimage
+from six.moves import xrange
```

Comment:
Ah sorry for that, I was in automatic diff processing mode and missed that completely, sorry for the noise ! I think it's fine to keep with xrange.